src/python_testing/TC_EVSE_2_1.py

```diff
@@ -0,0 +1,90 @@
+
+import logging
+
+import chip.clusters as Clusters
+from chip.interaction_model import Status
+from matter_testing_support import MatterBaseTest, async_test_body
+from mobly import asserts
+
+# List of attributes tests in the following format: ['Attribute name',read/write, Value to Write, ExpectedValue, Test step]
+Attribute_names = [
```

Comment:
So this feels fragile, like it's going to fall out of step quickly with the spec as it moves forward. That being said, I don't have a GREAT solution right now to fix this. I do have this PR up that will pull this info from the spec automatically: https://github.com/project-chip/connectedhomeip/pull/30772/files

It doesn't do ranges YET, but if this is something that would be helpful, it can be added.

I obviously can't block this PR on a pending PR that doesn't have all the stuff you need, but can you add a TODO for either of us to come back in here and update it once the DM XML markers for read/write and ranges lands? Then we don't have to worry about manual changes.

You'd still need to handle the out of bounds checks in a separate cluster test, thanks to the wishy-washy out of bounds handling in the spec, but at least this table generation would be automated.

nit: A dataclass might be better here. Given that I'm hoping this table will go away, I'm not sure I care though.